class.smtp.php

```diff
@@ -1189,4 +1192,17 @@ public function getTimeout()
     {
         return $this->Timeout;
     }
+
+    /**
+     * Reports an error number and string.
+     * @param integer $errno The error number returned by PHP.
+     * @param string $errmsg The error message returned by PHP.
+     */
+    function errorHandler($errno, $errmsg)
+    {
+        $this->edebug(
```

Comment:
This should call `setError` too. Otherwise If debugging is disabled, you'll never hear about the error - wasn't that the point of adding this?